sklearn/tests/test_cross_validation.py

```diff
@@ -479,24 +477,26 @@ def assert_counts_are_ok(idx_counts, p):
         assert_counts_are_ok(train_counts, ex_train_p)
         assert_counts_are_ok(test_counts, ex_test_p)
 
-def test_predefinedsplit_with_kfold_split():       
-    '''Check that PredefinedSplit can reproduce a split generated by Kfold.'''     
-    folds = -1 * np.ones(10)       
-    kf_train = []      
-    kf_test = []       
-    for i, (train_ind, test_ind) in enumerate(cval.KFold(10, 5, shuffle=True)):        
-        kf_train.append(train_ind)     
-        kf_test.append(test_ind)       
-        folds[test_ind] = i        
-    ps_train = []      
-    ps_test = []       
-    ps = cval.PredefinedSplit(folds)       
-    for train_ind, test_ind in ps:     
-        ps_train.append(train_ind)     
-        ps_test.append(test_ind)       
-    assert_array_equal(ps_train, kf_train)     
+
+def test_predefinedsplit_with_kfold_split():
```

Comment:
I think this is more due to usage of `\r` for newline instead of `\n` (I think)... see https://github.com/scikit-learn/scikit-learn/pull/4250#discussion_r25539805 ;) There I had used a script that writes to the files line by line appending `\n` at the end of each line...